modules/map/hdmap/hdmap_common.cc

```diff
@@ -308,8 +308,15 @@ double LaneInfo::GetWidthFromSample(
   }
   int low = 0;
   int high = static_cast<int>(samples.size());
+  int mid;
   while (low + 1 < high) {
-    const int mid = (low + high) / 2;
+    double tmp = samples[high].first - samples[low].first;
```

Comment:
Seems like `samples[samples.size()]` will out of bounds

Reply:
> Seems like `samples[samples.size()]` will out of bounds

Not "seems like",  high=N will surely make visit to the element out of bounds.